### DISCUSSION TITLE
Staging ISOs

### DISCUSSION THREAD

**Discussion by bunnymnemonic** [SpiralLinux/SpiralLinux-project#352] [Q&A] [answered]
Hi,
First of all, thank you very much for this excellent distribution!
Do you recommend using the staging version as a daily driver? Or do you plan to release a "stable" version of it?
Thank you for your answer in advance.

**Comment by geckolinux** [comment, marked as answer]
Hi there, they make work well, but they haven't been fully tested. An official release announcement is always made when new ISOs are formally released, and https://spirallinux.github.io is updated with the latest download links.

**Reply by bunnymnemonic** [reply]
Thank you for your answer!
